### PR TITLE
i18n(es): update view-transitions.mdx

### PR DESCRIPTION
**Pull request by AitorMT**
#### Description (required)

Commit based on the following pull request. #7049

### CODE REVIEW

File: src/content/docs/es/guides/view-transitions.mdx [hunk 1]
```diff
@@ -301,24 +301,25 @@ import { ViewTransitions } from "astro:transitions";
...
 export default function Form() {
-  return <select onChange={(ev) => navigate(e.target.value)}>
-    <option value="/play">Jugar</option>
-    <option value="/blog">Blog</option>
-    <option value="/about">Acerca de</option>
-    <option value="/contact">Contacto</option>
-  </select>;
+  return (
+    <select onChange={(e) => navigate(e.target.value)}>
+      <option value="/play">Play</option>
```
Comment (AitorMT) [inline src/content/docs/es/guides/view-transitions.mdx:310]:
```suggestion
      <option value="/play">Jugar</option>
```

File: src/content/docs/es/guides/view-transitions.mdx [hunk 2]
```diff
@@ -301,24 +301,25 @@ import { ViewTransitions } from "astro:transitions";
...
+      <option value="/blog">Blog</option>
+      <option value="/about">About</option>
```
Comment (AitorMT) [inline src/content/docs/es/guides/view-transitions.mdx:312]:
```suggestion
      <option value="/about">Acerca de</option>
```

File: src/content/docs/es/guides/view-transitions.mdx [hunk 3]
```diff
@@ -301,24 +301,25 @@ import { ViewTransitions } from "astro:transitions";
...
+      <option value="/contact">Contact</option>
```
Comment (AitorMT) [inline src/content/docs/es/guides/view-transitions.mdx:313]:
```suggestion
      <option value="/contact">Contacto</option>
```

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Pull request merged**